test/fixtures/terms.yml

```diff
@@ -13,3 +13,15 @@ cool:
 
 hi:
   phrase: hello world
+
+pmid_38908367:
+  phrase: 'TERT activation targets DNA methylation and multiple aging hallmarks. Shim HS, et al. Cell. 2024. PMID: 38908367'
+
+issn_1075_8623:
+  phrase: 1075-8623
+
+doi:
+  phrase: '10.1016/j.physio.2010.12.004'
+
+isbn_9781319145446:
+  phrase: 'Sadava, D. E., D. M. Hillis, et al. Life: The Science of Biology. 11th ed. W. H. Freeman, 2016. ISBN: 9781319145446'
```

Comment:
We definitely get traffic like that and anecdotally it's fine, but I agree it might be worth creating explicit tests for this behavior to make sure we don't unintentionally break it in the future.

Reply:
https://mitlibraries.atlassian.net/browse/TCO-43